pyaerocom/ungriddeddata.py

```diff
@@ -372,17 +372,18 @@ def from_station_data(stats, add_meta_keys=None):
 
     def add_station_data(self, stat, meta_idx=None, data_idx=None, check_index=False):
         raise NotImplementedError("Coming at some point")
-        if meta_idx is None:
-            meta_idx = self.last_meta_idx + 1
-        elif meta_idx in self.meta_idx:
-            raise ValueError(
-                f"Cannot add data at meta block index {meta_idx}, index already exists"
-            )
-
-        if data_idx is None:
-            data_idx = self._data.shape[0]
-        elif not np.all(np.isnan(self._data[data_idx, :])):
-            raise ValueError(f"Cannot add data at data index {data_idx}, index already exists")
+        # Do not remove below. Keep until implemented, but should not count against coverage.
+        # if meta_idx is None:
+        #     meta_idx = self.last_meta_idx + 1
+        # elif meta_idx in self.meta_idx:
+        #     raise ValueError(
+        #         f"Cannot add data at meta block index {meta_idx}, index already exists"
+        #     )
+
+        # if data_idx is None:
+        #     data_idx = self._data.shape[0]
+        # elif not np.all(np.isnan(self._data[data_idx, :])):
+        #     raise ValueError(f"Cannot add data at data index {data_idx}, index already exists")
```

Comment:
please remove dead code

Reply:
I specifically wrote "Do not remove below", because at this point, I do not know if this can be removed from the codebase. This section originlly appeared after `raises(NotImplementedError("Coming at some point"))`, but was counting against our code coverage. Until it is functional in the code, I do not believe it should count against our coverage. I can always leave it as I found it.

Reply:
sorry, I did not see the `Do not remove` part.
Can it be moved to a separate function that raises NotImplementedError as the 1st line?
That way the whole function can me marked with `pragma: no cover`

Reply:
That essentially what it originally was, except without the `pragma: no cover`. I added one and uncommented out the code again. That seems cleaner